install-guides/README.md

```diff
@@ -0,0 +1,191 @@
+- Ubuntu: /etc/default/kubelet
+- CentOS: /etc/sysconfig/kubelet
+
+ Replace the line KUBELET_EXTRA_ARGS with the following:
+
+```
+KUBELET_EXTRA_ARGS=\
+--cpu-manager-policy=static \
+--kube-reserved=cpu=1,memory=2Gi,ephemeral-storage=1Gi \
+--system-reserved=cpu=1,memory=2Gi,ephemeral-storage=1Gi \
+--topology-manager-policy=single-numa-node \
+--feature-gates="TopologyManager=true,KubeletPodResourcesGetAllocatable=true"
+
+```
+
+The next step is to remove the kubelet cpu checkpoint and restart the service.
+
+```
+systemctl stop kubelet
+systemctl daemon-reload
+rm -f /var/lib/kubelet/cpu_manager_state
+systemctl start kubelet
+```
+To check that the above has take effect run `systemctl status kubelet` and review the arguments passed to kubelet. The above args, including topology manager and cpu manager policies, should now be in place.
+
+### Install Topology Exporter
+In Kubernetes 1.21 a number of enhancements have been made to the Kubelet [to support Topology Scheduling](https://github.com/kubernetes/website/blob/64ef8768ab9ae25a5338a95225a80ac90423473f/content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md#monitoring-device-plugin-resources). These changes are enabled by the feature flag 'KubeletPodResourcesGetAllocatableTrue' and expose information about device topology on a gRPC service running locally on each node. A Resource Topology Exporter can then be used in order to read this information and make it available in the Kubernetes API.
+
+The Resource Topology Exporter is part of Node Feature Discovery. It reads information from the Kubelet's endpoint and creates a representation of the Kubernetes node as a result.
+
+To install Node Feature Discovery from the development branch:
+
+```
+git clone https://github.com/k8stopologyawareschedwg/node-feature-discovery.git
+cd node-feature-discovery
+git checkout k8stopologyawareschedwg:topology-updater-implementation 
+make image
+kubectl apply -f  manifests/noderesourcetopologies.yaml
+kubectl apply -f manifests/nfd-daemonset-master-topology-updater.yaml
+```
+
+Once Node Feature discovery is up and running it should begin reporting on the resources per numa node for the node.
+
+To see these descriptions run:
+```
+kubectl describe  noderesourcetopologies
+```
+
+### Install Topology Aware Scheduler
+
+At this point our kubelet is exposing information about resource Topology and our Node Feature Discovery agent is making this information available in the Kubernetes API. The next step is to schedule based on the per-node resource topology information.
+
+Node Resource Topology Scheduling is currently enabled in the [community sponsored scheduler plugins repo](https://github.com/kubernetes-sigs/scheduler-plugins/tree/release-1.19).  The image from the scheduler plugins repo has all of the scheduling power of the Kubernetes core scheduler with some added useful plugins - like Resource Topology Scheduling - built in on top.
```

Comment:
Please add this link https://github.com/kubernetes-sigs/scheduler-plugins/blob/master/pkg/noderesourcetopology/README.md here as well.

Reply:
👍